### PR TITLE
docs: Add HTTP port to collector docker command

### PR DESCRIPTION
**Pull request by kaylareopelle**
The docker run command includes a port for GRPC. If someone is using an HTTP OTLP exporter along with the collector, this command will not work out of the box. 

This commit adds the HTTP port to the docker run command.

Discovered by @wsmoak in a discussion on: https://github.com/open-telemetry/opentelemetry-ruby/issues/1525#issuecomment-2419583992

### CODE REVIEW

**Review by tiffany76 — APPROVED** [review]
LGTM. @open-telemetry/collector-approvers, PTAL. Thanks.

File: content/en/docs/collector/quick-start.md [hunk 1]
```diff
@@ -61,7 +61,8 @@ preferred shell.
 
    ```sh
    docker run \
-     -p 127.0.0.1:4317:4317 \
+     -p 127.0.0.1:4317:4317 \ # for gRPC
```
Comment (codeboten) [inline content/en/docs/collector/quick-start.md:64]: i think the comment here will cause problems

Comment (codeboten) [reply content/en/docs/collector/quick-start.md:64]:
```
$ docker run \
     -p 127.0.0.1:4317:4317 \ # for gRPC
     -p 127.0.0.1:4318:4318 \ # for HTTP
     -p 127.0.0.1:55679:55679 \
     otel/opentelemetry-collector-contrib
docker: invalid reference format.
See 'docker run --help'.
zsh: command not found: -p
zsh: command not found: -p
```

Comment (codeboten) [reply content/en/docs/collector/quick-start.md:64]: tried in bash as well to make sure

```
bash-3.2$ docker run \ -p 127.0.0.1:4317:4317 \ # for gRPC -p 127.0.0.1:4318:4318 \ # for HTTP -p 127.0.0.1:55679:55679 \ otel/opentelemetry-collector-contrib
docker: invalid reference format.
See 'docker run --help'.
```

**Review by chalin — CHANGES_REQUESTED** [review]
Thanks for this addition @kaylareopelle.

@codeboten, right, I'm not surprised that we can't have comments after end of line continuation characters.

All: this being a quick start, I'd suggest just dropping the port-use comments.

**Comment by kaylareopelle** [comment]
> > All: this being a quick start, I'd suggest just dropping the port-use comments.
> 
> Or even just above the docker run command:
> 
> ```
> ```sh
>    # 4317 for gRPC, 4318 for HTTP, 55679 for...
>    docker run \
>      -p 127.0.0.1:4317:4317 \
> ```

Great point, @codeboten! My bad for not testing the snippet with the comments before opening this PR. 

I updated the docs to place the comment at the top in 0b0a0d44. However, I'm also open to removing it entirely. 

Please take another look when you have a chance!

File: content/en/docs/collector/quick-start.md [hunk 2]
```diff
@@ -60,8 +60,10 @@ preferred shell.
 3. Launch the Collector:
 
    ```sh
+   # 4317 for gRPC, 4318 for HTTP, 55679 for ZPages
```
Comment (chalin) [inline content/en/docs/collector/quick-start.md:63]: I still feel it better to just drop the comment, but will let other @open-telemetry/docs-approvers chime in. If we keep it, let's be clear that this comment is referring to port numbers.

Comment (svrnm) [reply content/en/docs/collector/quick-start.md:63]: Can we move that comment into the text above, like "Launch the Collector, listening on ports 4317 (for OTLP grPC), 4318 (for OTLP HTTP) and 55679 (for ZPages):"

Comment (kaylareopelle) [reply content/en/docs/collector/quick-start.md:63]: Either way works for me! I updated the PR to use the text @svrnm suggested in: c9d08a4e

File: content/en/docs/collector/quick-start.md [hunk 3]
```diff
@@ -57,11 +57,12 @@ preferred shell.
 
 ## Generate and collect telemetry
 
-3. Launch the Collector:
+3. Launch the Collector, listening on ports 4317 (for OTLP grPC), 4318 (for OTLP HTTP) and 55679 (for ZPages):
```
Comment (kaylareopelle) [inline content/en/docs/collector/quick-start.md:60]:
```suggestion
3. Launch the Collector, listening on ports 4317 (for OTLP gRPC), 4318 (for OTLP HTTP) and 55679 (for ZPages):
```

**Comment by kaylareopelle** [comment]
I don't really understand what the `Files / FILE FORMAT (pull_request)` check is looking for. What needs to be changed to get it to pass?

**Comment by tiffany76** [comment]
Hi @kaylareopelle. The file formatting check flagged the line length: our linter looks for 80 characters per line. I've fixed it with a GitHub action. You should be all set now.

**Comment by kaylareopelle** [comment]
@chalin - Could you take another look? Since you requested changes, I need your approval before we can move forward.

**Review by dmitryax — APPROVED** [review]

**Pull request merged**